app/models/project.rb

```diff
@@ -61,7 +61,7 @@ def samples
   end
 
   def cleaned_project_name
-    "project-#{name.downcase.split(' ').join('_')}"
+    name.downcase.split(' ').join('_')
```

Comment:
The `project-` prefix seems unnecessary and just makes the name less readable. This cleaned project name is only being used in a couple of places, to generate the file name in a couple of single sample downloads.